samples/msal-angular-v2-samples/angular-b2c-sample-app/README.md

```diff
@@ -32,15 +32,16 @@ Open `.src/app/b2c-config.ts` in an editor:
 
 ### Running the sample
 
-- In a command prompt, run `npm start`.
+- Install the dependencies by running `npm install` in a terminal.
+- Once this is done, run `npm start`.
 - Navigate to [http://localhost:4200](http://localhost:4200)
 - In the web page, click on the "Login" button. The app will automatically reload if you change any of the source files.
 
 ### How to handle B2C user-flows
 
-Implementing B2C user-flows is a matter of initiating authorization requests against the corresponding authorities. This sample demonstrates [sign-up/sign-in](https://docs.microsoft.com/azure/active-directory-b2c/add-sign-up-and-sign-in-policy?pivots=b2c-user-flow) (with [self-service password reset](https://docs.microsoft.com/azure/active-directory-b2c/add-password-reset-policy?pivots=b2c-user-flow#self-service-password-reset-recommended)) and [edit-profile](https://docs.microsoft.com/azure/active-directory-b2c/add-profile-editing-policy?pivots=b2c-user-flow) user-flows.
+Implementing B2C user-flows is a matter of initiating authorization requests against the corresponding authorities. This sample demonstrates [sign-up/sign-in](https://docs.microsoft.com/azure/active-directory-b2c/add-sign-up-and-sign-in-policy?pivots=b2c-user-flow), legacy [password-reset](https://docs.microsoft.com/azure/active-directory-b2c/add-password-reset-policy?pivots=b2c-user-flow#password-reset-policy-legacy) and [edit-profile](https://docs.microsoft.com/azure/active-directory-b2c/add-profile-editing-policy?pivots=b2c-user-flow) user-flows.
```

Comment:
Out of curiosity, why are we demonstrating the legacy password reset flow here instead of self-service password reset if that is the recommended approach? Is the self-service password reset easier to implement?

Reply:
every now and then folks ask about how to handle legacy password reset, so I thought we could keep this one as reference and react b2c sample for the new flow. Also it would be good to do E2E tests with legacy flows since they are technically still supported.